### DISCUSSION TITLE
LambdaBuffers on Catalyst Fund10

### DISCUSSION THREAD

**Discussion by bladyjoker** [mlabs-haskell/lambda-buffers#101] [Announcements]
We issued a proposal for Catalyst Fund10 to get codegen support for Rust, Javascript and Aiken (https://cardano.ideascale.com/c/idea/105975/).

**Comment by bladyjoker** [comment]
I'm glad to announce that LambdaBuffers proposal got accepted this round as well!

Our Milestone document is being reviewed and work is already underway to get Rust supported as a target language. @szg251 is joining the team and will be taking on this challenge!

**Reply by euonymos** [reply]
Congrats to the team and happy hacking on LB!
